doc/manpages/ocne-config.yaml.md

```diff
@@ -112,6 +112,16 @@ clusterDefinition: mycluster.yaml
 clusterDefinitionInline: |
   somekey: someval
   someotherkey: someotherval
+  
+# This field specifies the password set for the "ocne" user.
+# This configuration is applied through ignition. Certain providers
+# require an ignition file to be passed in with the desired password
+# specified to enable log in. The "ocne" user is useful for obtaining the 
```

Comment:
Should it be "login" instead of "log in".  If so same if true for both files changed.